xbmc/win32/Win32DelayedDllLoad.cpp

```diff
@@ -70,5 +70,5 @@ FARPROC WINAPI delayHookFailureFunc (unsigned dliNotify, PDelayLoadInfo pdli)
 }
 
 // assign hook functions
-PfnDliHook __pfnDliNotifyHook2 = delayHookNotifyFunc;
-PfnDliHook __pfnDliFailureHook2 = delayHookFailureFunc;
```

Comment:
PR title says it moves sources ... well seems we shouldn't trust each other that much then ;)